contracts/AdjudicationFramework/MinimalAdjudicationFramework.sol

```diff
@@ -76,6 +76,7 @@ contract MinimalAdjudicationFramework {
     // When they're all cleared they can be unfrozen.
     mapping(address => uint256) public countArbitratorFreezePropositions;
 
+    uint256 public arbitrationDelayForCollectingEvidence;
```

Comment:
Just on the variable name I wouldn't be this prescriptive, you can use the time for discussion as well as collecting evidence

Reply:
I don't understand the stuff about restricting things to the payer. If the thing can be cancelled, you can't allow the payer to leave it in the pending state because nobody else can use it and everyone's funds are stuck. Anyone should be able to cancel something that isn't going to go through.

Reply:
If we don't do it, then anyone can reinitiated the forking for the payer, even if they don't want it to. Anyone can just call the funciton requestActivateFork and it would be reinitated or if the payer would like to request it, anyone could just send it back.

when are funds stuck? please explain.

Reply:
I mean that if this is happening at the end of a reality.eth game, you've got a load of bonds in the reality.eth contract, then potentially a whole chain of things waiting for that to resolve. So the payer shouldn't be able to make the thing take an arbitrarily long time.

If the desired behaviour is that the payer should decide whether to cancel or retry, the obvious thing to do is to give them a day or so then let anyone else do it after that. I think we used that pattern elsewhere. Or for now it might be simpler just to do the cancel route, it's the same result but with a few extra clicks.

Reply:
Okay, we can do the delay thing. I thought it might external contract, but happy to build it into it here, then its safer and enforced. But this will be another PR. THis PR is about the delay only.

Reply:
create an issue: https://github.com/RealityETH/subjectivocracy/issues/199